microshift_getting-started/microshift-architecture.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="microshift-architecture"]
+= Architecture
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-architecture
+
+toc::[]
+
+Determine which version of {product-title} you have, find out about known issues, and troubleshoot common problems.
```

Comment:
This looks like the same intro that appears in the troubleshooting page. Is that what we want?

Reply:
Nope, that's a mistake, good catch.